ITISFoundation/osparc-simcore
# Design improvements

## What do these changes do?


* Adding a thumbnail to the Node entity. This is being tested in 2D multi plots.
* Removed progress bars in project browser.
* Changed NodeUI design. Borders removed, simplified layout.
* Redesign ControlsBar.
* Workbench links misplacing bug fixed.
* Removed SideBar splitter (this is waiting for a Qooxdoo update, so meanwhile will be feature toggled till that's available).
* Sidebar is now collapsible.

## Related issue number


connects to #288 #671 

## Test it
* Create a 2D Multi plot and check the view in the main study workbench and in the specific internal 2D multi view.
* Check new NodeUI design, fancy it? Suggestions welcomed. Nodes will probably get bigger with the addition of tags, in the near future.
* Bottom bar with simulation controls, now takes full width, it will be used for other controls, like the node filtering abilities.
* Workbench links now point always to the same pixel of the node.
* Side bar splitter is removed but it will actually be back until the modifications in Qooxdoo will be done. I added a feature toggle if to avoid errors. As a reminder, a console.warn was added.
* Play around collapsing/uncollapsing  and resizing the sidabar, making sure it doesn't bug.

## Checklist


## Captures
![Screenshot from 2019-03-28 13-48-18](https://user-images.githubusercontent.com/4764217/55158878-32a69900-5160-11e9-8b20-035c129a305f.png)
![collapse-sidebar](https://user-images.githubusercontent.com/4764217/55384896-2e95c500-552c-11e9-811b-cd9f7594473b.gif)


[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet
